js/views/conversation_view.js

```diff
@@ -341,16 +341,13 @@
       this.selectMember = this.selectMember.bind(this);
 
       const updateMemberList = async () => {
-        const maxToFetch = 1000;
-        const allMessages = await window.Signal.Data.getMessagesByConversation(
-          this.model.id,
-          {
-            limit: maxToFetch,
-            MessageCollection: Whisper.MessageCollection,
-          }
+        const allMembers = this.model.messageCollection.models.map(d =>
```

Comment:
With this change, do we only consider the messages already in memory, e.g. only the 50 messages shown and not up to 1000 as it was before?

Reply:
Yeah that's what I wasn't sure. I think the default limit is 100, but still, bummer.